web/components/view/ShowHideTextView/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import arrow from '../../../assets/arrow.svg';
+
+class ShowHideTextView extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {isToggleOn: true};
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState(state => ({
+      isToggleOn: !state.isToggleOn
+    }));
+  }
+
+  render() {
+    return (
+      <div className="show-hide card secondary-outline">
+        <div className="pad-sides-s pad-ends-s user-select-none clickable flex jc-between" onClick={this.handleClick}>
+          <label className="show-hide-outer-text margin-ends-s margin-right-s clickable">{this.props.outerText}</label>
+          <img className="margin-ends-s flex-no-shrink" src={arrow} alt="arrow"/>
+        </div>
+        <p className={`show-hide-inner-text margin-sides-s margin-bottom-s ${this.state.isToggleOn ? 'display-none' : ''}`}>
```

Comment:
could we add transition animation? wouldn't mind it being in another pr tho

Reply:
I was thinking a transition would be nice, but I don't think this is as easy as adding a fade like in the other one - I think another PR for that would be best

Reply:
Agreed, I'll backlog this part ^